packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBarRow.js

```diff
@@ -86,12 +87,15 @@ export default class SpanBarRow extends React.PureComponent<SpanBarRowProps> {
       numTicks,
       rpc,
       showErrorIcon,
+      getViewedBounds,
+      traceStartTime,
       span,
-      viewEnd,
-      viewStart,
     } = this.props;
     const { duration, hasChildren: isParent, operationName, process: { serviceName } } = span;
     const label = formatDuration(duration);
+    const viewBounds = getViewedBounds(span.startTime, span.startTime + span.duration);
+    const viewStart = viewBounds.start;
+    const viewEnd = viewBounds.end;
```

Comment:
does this account for a possible zoom on the minimap? I.e. do we test with less than 100% minimap selection?

Reply:
Yes, it accounts for a zoom.